docs/developer-guide/internationalization.md

```diff
@@ -48,6 +48,43 @@ Setting locales in localConfig.json file is doable only for supported locales pr
 The default behavior is to use those already configured in "supportedLocales" object.
 You can customize the messages by editing the data.code-CODE.json files.
 
+You can also add the `defaultUserLocale` property to localConfig.json to override the initial user locale:
```

Comment:
```suggestion
You can also add the `defaultUserLocale`, the locale **code**,  property to localConfig.json to override the initial user locale:
```

Reply:
If I understood correctly, it forces the initial locale. if so, maybe is not properly `defaultUserLocale` but `forceUserLocale` , `initialLocale` or something similar, what do you thing @allyoucanmap

Reply:
@MV88 @offtherailz I called `defaultUserLocale` because we use the function `getUserLocale()` so I did not think much about this. In my opinion `initialLocale` or `defaultLocale` could work. Let me know which one you prefer.